### ISSUE TITLE
Is there a planned release schedule?

### ISSUE THREAD

**Issue by yzgyyang** [Zulko/moviepy#1566]
Hi! Just wondering if you plan to make newer releases anytime soon. The latest 1.x release is over a year old, and the 2.0 dev release is also half a year ago - IMO there are enough bugfixes and features that we can make use of from a new release!

(I am aware of #1089, but that hasn't had an update for a while as well)

Thanks again for all the hard work!

**Comment by tburrows13** [comment]
Indeed, it is long overdue! The main issue is that because it 2.0 has breaking changes in it, the plan is too bundle as many together as possible to minimise disruption, but it wasn't supposed to take this long. The main thing left is the textclip change discussed in #1472. I've not had time recently to further this and other issues, but I will have over the summer, so if nothing has advanced there by ~July then I shall finish that bit myself.

At that point, there'll be a final pre release and then a release a week or 2 later.

This is my current plan, I'm very happy to take feedback on it. The main issue is maintainer time, if no one is working on the project, then it cannot advance! Because of this, release schedules are rather pointless. If work is done, then it can be released (outside of the breaking-change-problem mentioned above).

**Comment by OsaAjani** [comment]
Thank you for your contributions and for reporting issues in this repository. With the release of v2, which introduces significant changes to the codebase and API, we’ve reviewed the backlog of open PRs and issues. Due to the length of the backlog and the likelihood that many of these are either fixed or no longer applicable, we’ve made the decision to close all previous PRs and issues.

If you believe that any of these are still relevant to the current version or if you'd like to reopen a related discussion, please feel free to create a new issue or pull request, referencing the old one.

Thank you for your understanding and continued support!
